app/src/main/java/com/asuc/asucmobile/domain/main/PopUpActivity.java

```diff
@@ -73,19 +80,30 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 icon.setImageResource(R.drawable.sleeppod_map_icon);
                 markerTitle.setTextColor(getResources().getColor(R.color.pink));
                 break;
+            case "Ford Go Bike":
```

Comment:
The strings  are defined in `values` for reusability and consistency

Reply:
@rustielin will change this too